.nais/naiserator-dev.yaml

```
@@ -42,6 +42,7 @@ spec:
   gcp:
     sqlInstances:
       - type: POSTGRES_14
+        tier: db-f1-micro
         databases:
           - name: isoppfolgingstilfelle-db
         diskAutoresize: true
```

Det blir kanskje akkurat det samme? Dette var løsningsforslaget jeg fikk fra Joakim 🤔 Forskjellen er kanskje at når man har disse propertiene utenfor så gjelder det alle databaser, imens når de er indentert så gjelder det den spesifikke? 😊
```suggestion
  gcp:
    sqlInstances:
      - type: POSTGRES_14
        databases:
          - name: isoppfolgingstilfelle-db
             tier: db-f1-micro
             diskAutoresize: true
```

---

Trooor det må være utenfor, ref https://doc.nais.io/workloads/application/reference/application-spec/#gcpsqlinstancestier og [eksempelet](https://doc.nais.io/persistence/postgres/). Er sånn vi har det i andre apper også.